nflow-engine/src/main/java/io/nflow/engine/internal/dao/WorkflowInstanceDao.java

```diff
@@ -676,13 +684,13 @@ public int compareTo(OptimisticLockKey other) {
     }
     if (query.includeChildWorkflows) {
       for (final WorkflowInstance instance : ret) {
-        fillChildWorkflows(instance);
+        fillChildWorkflowIds(instance);
```

Comment:
We could still support getting e.g. child workflows or only their ids without cluttering the method signature if we had ```WorkflowInstanceEnrichment``` (see previous comment).